js/graph_spectrum_plot.js

```diff
@@ -340,37 +360,40 @@ GraphSpectrumPlot._drawFiltersAndMarkers = function(canvasCtx) {
         console.log('Notch filter fieldName missing');
     }
 
-    if (this._spectrumType == SPECTRUM_TYPE.FREQUENCY) {
-        this._drawInterestFrequency(canvasCtx, this._fftData.maxNoiseIdx,  PLOTTED_BLACKBOX_RATE, 'Max motor noise', WIDTH, HEIGHT, (15*offset) + MARGIN, "rgba(255,0,0,0.50)", 3);
+    if (this._spectrumType === SPECTRUM_TYPE.FREQUENCY) {
+        this._drawInterestFrequency(canvasCtx, this._fftData.maxNoiseIdx,  PLOTTED_BLACKBOX_RATE, 'Max motor noise',
+                                    WIDTH, HEIGHT, (15 * offset) + MARGIN, "rgba(255,0,0,0.50)", 3);
+        offset++;
     }
 
 };
 
 GraphSpectrumPlot._drawNotCachedElements = function() {
 
-    var canvasCtx = this._canvasCtx; // Not cached canvas
+    const canvasCtx = this._canvasCtx; // Not cached canvas
 
     canvasCtx.save();
 
-    var PLOTTED_BLACKBOX_RATE = this._fftData.blackBoxRate / this._zoomX;
+    const PLOTTED_BLACKBOX_RATE = this._fftData.blackBoxRate / this._zoomX;
 
-    var HEIGHT = canvasCtx.canvas.height - MARGIN_BOTTOM;
-    var WIDTH  = canvasCtx.canvas.width - this._getActualMarginLeft();
-    var LEFT   = canvasCtx.canvas.offsetLeft + this._getActualMarginLeft();
-    var TOP    = canvasCtx.canvas.offsetTop;
+    const HEIGHT = canvasCtx.canvas.height - MARGIN_BOTTOM;
+    const WIDTH  = canvasCtx.canvas.width - this._getActualMarginLeft();
```

Comment:
Again, 'fancy' formatting - if `FONT_SIZE` is added here, there will be 5 lines to edit / review instead of one - not what I think we should be doing.